pytentiostat/routines.py

```diff
@@ -38,14 +38,16 @@ def _initialize_arduino(com):
     Creates board object with Arduino(). If the connection fails it prints an error message and exits.
     Parameters
     ----------
-    com: string
```

Comment:
Not sure why this part of doc string is deleted.

Reply:
Sorry, that was an error.

Reply:
that was a mistake